revert_deploy.yml

```diff
@@ -0,0 +1,55 @@
+###
+#  Set to previous deploy in case of an error or problem.
+#
+# Example usage:
+# ansible-playbook -e host_group=prosody_qa revert_deployment.yml
+#
+# The script will ask for confirmation before making any breaking changes.
+# ctrl+c and then 'a 'will break execution, enter or ctrl+c and then 'c' will
+# proceed.
+#
+# NOTE: You will need the password file loaded for any secrets encrypted in
+# the vaults used by the host_group specified.
+###
+
+- hosts: '{{ host_group }}'
+  connection: ssh
+  remote_user: deploy
+  environment:
+    PATH: '{{ path }}'
+    LD_LIBRARY_PATH: '{{ ld_library_path }}'
+    PYTHONPATH: '{{ python_path }}'
+  tasks:
+  - name: Pause and let user make sure command looks right.
+    pause:
+      prompt: >
+               You are about to revert {{ host_group | upper }}.
+               Press enter to proceed or ctrl+c then a to abort.
```

Comment:
Do you think we really need confirmation when it's a separate command to run revert? Seems like it would be unlikely to happen accidentally.

Reply:
I think my innate caution with anything that could touch prod was at play here. My concern is someone inadvertently specifying say `mep` when they meant `mep_qa` (for example). If you're comfortable removing it, I'm comfortable removing it, just knowing we need to aim well.

Reply:
It seems less likely to me that we're going to revert qa very often - since we're usually ok with it being broken (at least briefly), and probably going to investigate and figure out a proper fix anyway.  I don't think we need a secondary confirmation.

@thatbudakguy any opinion?

Reply:
i feel like i generally run these commands with sufficient anxiety/paranoia such that confirmation prompt isn't super necessary, but fine either way

Reply:
All right, in that case I've dropped it.